src/Microsoft.Health.Dicom.Api.UnitTests/Features/Routing/UrlResolverTests.cs

```diff
@@ -86,12 +86,13 @@ public void GivenAStudy_WhenRetrieveStudyUriIsResolved_ThenCorrectUrlShouldBeRet
                 });
         }
 
-        [Fact]
-        public void GivenAStudy_WhenRetrieveStudyUriWithPartitionIdAndVersionIsResolved_ThenCorrectUrlShouldBeReturned()
+        [Theory]
+        [InlineData("v1.0-prerelease")]
+        [InlineData("v1.0")]
```

Comment:
let's change these to `"v1"` for consistency

Reply:
Updated.